packages/react-instantsearch-core/src/types/algoliasearch.ts

```diff
@@ -1,16 +1,15 @@
 // Custom types to support both algoliasearch
 // `v3` and algoliasearch `v4` clients.
 
-import type algoliasearch from 'algoliasearch';
-import type {
-  // @ts-ignore
-  Response as SearchResponseV3,
-} from 'algoliasearch';
+import type algoliasearch from 'algoliasearch/lite';
+import type * as AlgoliaSearch from 'algoliasearch/lite';
```

Comment:
```suggestion
import type algoliasearch, { Response as AlgoliaSearchResponse } from 'algoliasearch/lite';
```
Can't we merge this two imports?

Reply:
Got it, thanks!